tests/ui-toml/nonstandard_macro_braces/conf_nonstandard_macro_braces.rs

```diff
@@ -44,7 +44,7 @@ fn main() {
     let _ = format!["ugh {} stop being such a good compiler", "hello"];
     let _ = quote!(let x = 1;);
     let _ = quote::quote!(match match match);
-    let _ = test!();
+    let _ = test!(); // don't trigger for macro calls inside macros
```

Comment:
You know if the macro is defined within the same crate..... I would argue it makes sense to still emit the lint, but maybe it's not worth complicating the logic further.

Reply:
I think I got something that works, that was a bit of a rabbit hole, `Span` is a complicated thing...